tools/benchmark_tool/setup.py

```diff
@@ -21,6 +23,13 @@
         in pkg_resources.parse_requirements(requirements_txt)
     ]
 
+try:
+    importlib.import_module('cv2')
+except ImportError as opencv_import_error:
+    warnings.warn(
+            "Problem with cv2 import: \n{}\n opencv-python-headless will be added to requirements".format(opencv_import_error)
```

Comment:
May be better to say: `will be installed`

Reply:
we do not install it in this place, just add to requirements file and keep user informed that we do it

Reply:
P.S. Please don't make me change this if there is not strong reason as I will have to align it with the message inside AC (merge one more PR in omz), update submodule and then again ,create PR for releases/2022/2. It is painful for me and require a lot of time for precommit checks :)